include/oxen/quic/loop.hpp

```diff
@@ -98,22 +98,28 @@ namespace oxen::quic
         }
 
       private:
-        std::list<std::weak_ptr<Ticker>> _tickers;
+        static constexpr uint16_t loop_id{0};
```

Comment:
Would be better to use some typedef like `eventloop_id` here and in Network to clarify the purpose.

Reply:
Yep good call